src/Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions/IConnectionInformation.cs

```diff
@@ -12,9 +12,10 @@ public interface IConnectionInformation
         IPEndPoint LocalEndPoint { get; }
 
         PipeFactory PipeFactory { get; }
+        IScheduler InputReaderScheduler { get; }
```

Comment:
Undo these changes and leave the default dispatching to the thread pool. That dispatching to the application can be controlled via another flag directly in kestrel itself.